code/math/floating.h

```diff
@@ -35,6 +35,8 @@ float frand_range(float min, float max);
 #define flceil(fl) (int)ceil(fl)
 #define flfloor(fl) (int)floor(fl)
 #define f2fl(fx) ((float)(fx)/65536.0f)
+// more precise version of f2fl(), to double instead of float
+#define f2fld(fx) (((double)(fx))/65536.0)
```

Comment:
`f2fl` stands for "fix to float", so I would recommend naming the new macro `f2d` and removing the comment.